packaging/docker/README.md

```diff
@@ -103,21 +135,15 @@ services:
 
 Since we use an [ENTRYPOINT](https://docs.docker.com/engine/reference/builder/#entrypoint) directive, you can provide [Netdata daemon command line options](https://docs.netdata.cloud/daemon/#command-line-options) such as the IP address Netdata will be running on, using the [command instruction](https://docs.docker.com/engine/reference/builder/#cmd). 
 
-## Install Netdata using Docker Compose with SSL/TLS enabled http proxy
-
-For a permanent installation on a public server, you should [secure your Netdata instance](../../docs/netdata-security.md). This section contains an example of how to install Netdata with an SSL reverse proxy and basic authentication.
+## Install Netdata using Docker Compose with SSL/TLS enabled HTTP Proxy
 
-You can use use the following docker-compose.yml and Caddyfile files to run Netdata with docker. Replace the Domains and email address for [Letsencrypt](https://letsencrypt.org/) before starting.
+For a permanent installation on a public server, you should [secure the Netdata instance](../../docs/netdata-security.md). This section contains an example of how to install netdata with an SSL reverse proxy and basic authentication.
 
-### Prerequisites
-
--   [Docker](https://docs.docker.com/install/#server)
--   [Docker Compose](https://docs.docker.com/compose/install/)
--   Domain configured in DNS pointing to host.
+You can use the following docker-compose.yml and Caddyfile files to run Netdata with Docker. Replace the Domains and email address for [Let's Encrypt](https://letsencrypt.org/) before starting.
```

Comment:
Let's use an inline code block for the docker-compose file, since you're referencing a specific file:

```suggestion
You can use the following `docker-compose.yml` and Caddyfile files to run Netdata with Docker. Replace the domains and email address for [Let's Encrypt](https://letsencrypt.org/) before starting.
```